apps/docs/docs/state/effects/effects.mdx

```diff
@@ -342,108 +937,119 @@ export class FooComponent {
 }
 ```
 
-## Install
+## Case Study: Refactor to RxEffects
```

Comment:
I'm super sorry to tell that, but I don't feel the main documentation page is the right one for anything not directly related to main APIs and basic usage documentation. IMO, cut it out, create separate recipes page and reference sections in this page. Just feels as too much things on one page.